students/17th/team3/jaeyiahn/user/views.py

```diff
@@ -0,0 +1,45 @@
+import json
+
+from django.views import View
+from django.http  import JsonResponse
+
+from .models import User
+
+#회원가입
+class SignUpView(View):
+    def post(self, request):
+        try:
+            data     = json.loads(request.body)
+
+            username = data.get('username')
+            email    = data.get('email')
+            phone    = data.get('phone')
+            password = data.get('password')
+
+            if User.objects.filter(email=data['email']).exists():
+                return JsonResponse({'message': '중복된 email'}, status=400)
```

Comment:
```suggestion
                return JsonResponse({'message': 'EMAIL_EXISTS'}, status=400)
```
우리가 리스판스시 응답메세지를 주는부분은 사실 status_code 만으로도 충분합니다. 
하지만 보다 원할한 소통을 하고자 함으로서 메세지도 응답코드화해서 사용하는데, 이는 코드화된 문자열이어야 합니다. 
따라서 위에 작성하신것과 다르게 단순한 코드형태로 작성해서 프론트엔드와 소통하는 것이 좋습니다. 
결국 모든것은 규격화해서 전달하는게 좋습니다. 
아래의 다른 리턴도 변경 부탁드립니다.

Reply:
네 수정하겠습니다~